### DISCUSSION TITLE
Is diesel-async still considered experimental?

### DISCUSSION THREAD

**Discussion by Tuetuopay** [weiznich/diesel_async#130] [Q&A] [answered]
Everything is in the title. When first announcing diesel-async, it was said that it was experimental hence why at version 0.x and not under the diesel github organization. Basically the same question as https://github.com/weiznich/diesel_async/discussions/62 but for 0.4 almost a year later :)

Is this still the case, or it is now production-ready? (ignoring api breakages)

Thanks!

**Comment by weiznich** [comment, marked as answer]
I wouldn't consider this crate highly experimental anymore, but I still expect that larger changes will happen to the code. I won't judge whether or not it's "production" ready.  That's up to you as that really depends on your needs.

**Reply by Tuetuopay** [reply]
thank you for the swift answer!

> but I still expect that larger changes will happen to the code

do you have an idea on which areas per chance, or it's more a "I know it'll happen so be warned" statement?

thanks!

**Reply by weiznich** [reply]
There are still missing language features for fundamental async expressions in rust. I suppose that I want to use them as soon as they become stable, which in turn might require changes to the crate.
